### ISSUE TITLE
setOptions not working

### ISSUE THREAD

**Issue by huuvu1109** [seregazhuk/php-pinterest-bot#190]
Dear @seregazhuk ,

Can you please check why  "setOptions" is not working now. 

$bot->getHttpClient()->setOptions([
	CURLOPT_PROXY => $proxy,
	CURLOPT_USERAGENT => 'Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36',

	]);
	$var = $bot->user->registerBusiness($email, $pass, $rname);

I tried to set proxy and then try to registerBusiness() to create accounts and there are errors as follows:

(I have checked the proxy by Firefox and it works. Also, if I remove the setOptions() instructions, it works fine and can create new accounts)

Catchable fatal error: Argument 2 passed to seregazhuk\PinterestBot\Api\Response
::getValueByKey() must be of the type array, null given, called in E:\xampp\htdo
cs\pinterestNewVersionClientInfo\vendor\seregazhuk\pinterest-bot\src\Api\Respons
e.php on line 80 and defined in E:\xampp\htdocs\pinterestNewVersionClientInfo\ve
ndor\seregazhuk\pinterest-bot\src\Api\Response.php on line 94

**Comment by seregazhuk** [comment]
Hi, @huuvu1109 
What type of proxy is it? HTTP or SOCKS?
It seems like an empty response from the server, and library expects an array. I'll fix this issue with failing on empty responses, but it is not related with proxy. Maybe you miss some options? For example:

```php
$bot->getHttpClient()->setOptions([
    CURLOPT_PROXY => $proxy,
    CURLOPT_PROXYTYPE => CURLPROXY_HTTP // or CURLPROXY_SOCKS5
]);
```

**Comment by huuvu1109** [comment]
Yes,  I add CURLOPT_PROXYTYPE option and it works. But if I add wrong option between  CURLPROXY_HTTP and CURLPROXY_SOCKS5, the program stops because 

"Catchable fatal error:.  Argument 2 passed to seregazhuk\PinterestBot\Api\Response
::getValueByKey() must be of the type array, null given, .....|


I have tried to put it in try{} catch{} but that error still occur and stop the program. Can you change that? I mean if not successful, it throw an exception and I can handle base on the exception (e.g ignore in this case)

**Comment by seregazhuk** [comment]
Fixed. Now it will return an empty response if request failed.

**Comment by huuvu1109** [comment]
Hi @seregazhuk ,

It works now. Thanks!

**Comment by huuvu1109** [comment]
Hi @seregazhuk ,

When we open a Pin, Pinterest, beside Like, Pinterest also have one more option is Tried it like the following Figure, is there any way to do it with the bot in the current version? 

![image](https://user-images.githubusercontent.com/9253443/27244244-f47a3f6c-52e5-11e7-8713-e1678d8e4d6f.png)

**Comment by seregazhuk** [comment]
Hi, @huuvu1109 it is not yet available. I think next week will be a new release with this feature.

**Comment by seregazhuk** [comment]
Have no idea. But you can use this hash as a pin id, why do you need exactly a real pin id?

**Comment by seregazhuk** [comment]
@huuvu1109 I've just published a new release (5.4.4) and it has *try it* functionality. Check the [docs](https://github.com/seregazhuk/php-pinterest-bot#tryit) for examples.
